services/logger.js

```diff
@@ -15,33 +17,41 @@ const createLogger = () => {
   return createDevLogger(logLevel);
 };
 
-const createProdLogger = (logLevel) => {
+const createProdLogger = (currentLogLevel) => {
   //TODO: read log destination from env. (QA/Prod might be different destination, but log the same)
   //TODO: implement a real Prod logger
 
   const debugLogFunction = (message) => {
-    if (logLevel !== "debug") return;
+    const loggableLevels = ["debug"];
```

Comment:
I think that this pattern makes this file much less readable, turning 1 easy to glance through line into 3. Is there a reason for this change?

Reply:
+1

Reply:
also, we're planning to migrate this file back to typescript later on, which will make this easier to read.

Reply:
In the case of this specific one, with just one level, I agree it doesn't make it more clear.
But I changed because there was a bug on the other logs and I thought having it more verbose would avoid it easier.

The bug identified was this:
```
if (logLevel !== "debug" || logLevel !== "default") return;
```
The previous line above was evaluating to always true.

I can revert all changes and just change to `&&` instead if it's better.

Reply:
I reverted back, but kept the bug fix:
```
if (logLevel !== "debug" && logLevel !== "default") return;
```

Reply:
For the record, I didn't mind your *allow* focused approach; that was a pretty good idea for self-documentation. I just didn't think it needed to be 3 lines =). I'd be ok with something like `if ( ! ["debug","default"].includes(logLevel) ) return;` or a set mapping of these existing somewhere.

Thanks for reverting back. In the case of this logic change, @Ermish is this what you intended?

Reply:
looks good now!

Reply:
Awesome, thanks!